backend/lcfs/web/api/redis/views.py

```diff
@@ -1,44 +0,0 @@
-from fastapi import APIRouter
```

Comment:
Are we going to need these endpoints in the future?

Reply:
no, these were just placeholders left over from the inital app template